articles/connections/passwordless/guides/best-practices.md

```diff
@@ -0,0 +1,22 @@
+---
+title: Passwordless Connections Best Practices
+description: Passwordless Connections Best Practices
+toc: true
+topics:
+    - connections
+    - passwordless
+    - authentication
+    - concept
+---
+
+##  Linking Accounts
+
+When a user authenticates via Passwordless, the user is attached to the connection using Auth0 as the Identity Provider (IdP). Since you can't force users to use the same mobile phone number or email address every time they authenticate, users may end up with multiple user profiles in the Auth0 datastore; you can link multiple user profiles through [account linking](/extensions/account-link).
```

Comment:
```suggestion
When a user authenticates via Passwordless, the user is attached to the connection using Auth0 as the Identity Provider (IdP). Since you can't force users to use the same mobile phone number or email address every time they authenticate, users may end up with multiple user profiles in the Auth0 datastore; you can link multiple user profiles through [account linking](/link-accounts).
```